Week1/homework/app.js

```diff
@@ -135,20 +82,48 @@
       'https://media.newyorker.com/photos/590953e42179605b11ad3bbc/master/w_1298,c_limit/sarmadaimage.jpg',
   };
 
-  function bookCovers() {
-    for (const book in myBooks) {
-      // Create an image tage for the covers and link it to book key in myBooks object. +  styling the image width.
-      const cover = document.createElement('img');
-      cover.style.width = '350px';
-      cover.setAttribute('src', booksCoverObject[book]);
+  // Create and append unorderd ul & li
+  function generateList() {
+    const createUl = document.createElement('ul');
+    document.body.appendChild(createUl);
+
+    for (let i = 0; i < bookTitles.length; i++) {
+      const createLi = document.createElement('li');
+      createLi.setAttribute('id', bookTitles[i]);
+      createUl.appendChild(createLi);
+    }
+  }
+
+  function generateBookCards() {
+    // Add Title
+    for (let i = 0; i < bookTitles.length; i++) {
+      const createHeading = document.createElement('h3');
+      document.getElementById(bookTitles[i]).appendChild(createHeading);
+      const bookInfo = Object.keys(myBooks);
```

Comment:
This `bookInfo` constant is not needed. You can do `bookTitles[i]` to get the object that holds the info of the book, just like you did on the line above.

Reply:
Thanks for your helpful feedback. 
I have modified the code, and updated some links to improve the quality of the cover images. 

 - Please ignore the debugging sample files.